### PR TITLE
Qebhwt workflow now saves the adviser message

### PR DESCRIPTION
**Pull request by saisankargochhayat**
## Related Issues and Dependencies
Related issue - https://github.com/thoth-station/thoth-application/issues/511

### CODE REVIEW

File: qeb_thamos_advise.py [hunk 1]
```diff
@@ -97,17 +99,24 @@ def qeb_hwt_thamos_advise() -> None:
...
+            raise ValueError(f"Unknown configuration option for requirements format: {requirements_format!r}")
+
+        pipfile = project.pipfile.to_string()
+        pipfile_lock_str = project.pipfile_lock.to_string() if project.pipfile_lock else ""
+        application_stack = PythonStack(
+            requirements=pipfile, requirements_lock=pipfile_lock_str, requirements_format=requirements_format
         )
-        _LOGGER.info("Successfully submitted thamos advise call: %r", analysis_id)
     except Exception as exception:
         if isinstance(
```
Comment (saisankargochhayat) [inline qeb_thamos_advise.py:121]: @pacospace do we need this exception handling nowhere?

Comment (pacospace) [reply qeb_thamos_advise.py:121]: not anymore, we can remove it, thanks!

Comment (saisankargochhayat) [reply qeb_thamos_advise.py]: @pacospace thanks, I have removed the checks for the exception class.

File: qeb_thamos_advise.py [hunk 2]
```diff
@@ -97,37 +96,53 @@ def qeb_hwt_thamos_advise() -> None:
...
+        elif requirements_format in ("pip", "pip-tools", "pip-compile"):
+            project = Project.from_pip_compile_files(allow_without_lock=True)
         else:
-            _LOGGER.debug(json.loads(exception.body)["error"])  # type: ignore
-            exception_message = json.loads(exception.body)["error"]  # type: ignore
+            raise ValueError(f"Unknown configuration option for requirements format: {requirements_format!r}")
 
+        pipfile = project.pipfile.to_string()
+        pipfile_lock_str = project.pipfile_lock.to_string() if project.pipfile_lock else ""
+        application_stack = PythonStack(
```
Comment (pacospace) [inline qeb_thamos_advise.py:114]: the input to AdviserTriggerMessage is not PythoStack type but dict: https://github.com/thoth-station/user-api/blob/13e877a51fcff47336a4a319c2b486f268ff54d3/thoth/user_api/api_v1.py#L297 we use the method used in User-API here we do not talk to user-API

Comment (saisankargochhayat) [reply qeb_thamos_advise.py:114]: I am passing the type dict now.

**Review by pacospace — APPROVED** [review]
Thanks!!

**Pull request merged**